.github/workflows/ci.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
         run: melos build
 
       - name: Run TICS analysis
-        if: github.event_name == 'push'
+        # if: github.event_name == 'push'
```

Comment:
But does in the other cases have access to the TICSAUTHTOKEN?

Reply:
apparently it does - I just opened this PR to trigger a test run, which seems to have worked